salt/states/virt.py

```diff
@@ -239,6 +239,8 @@ def powered_off(name, connection=None, username=None, password=None):
 def running(name,
             cpu=None,
             mem=None,
+            os_type=None,
```

Comment:
Same comments in this file as in the module above: The arguments need to be moved down and the versionadded tags should be `Neon` instead of Fluorine.

Reply:
Fixed them all.